parent/pom.xml

```diff
@@ -1721,27 +1722,32 @@
                 <artifactId>jaxb-api</artifactId>
                 <version>${version.org.wso2.orbit.javax.xml.bind}</version>
             </dependency>
-	<dependency>
-		<groupId>org.osgi</groupId>
-		<artifactId>org.osgi.service.component.annotations</artifactId>
-		<version>${version.org.osgi.service.component.annotations}</version>
-	</dependency>
-	<dependency>
-		<groupId>org.osgi</groupId>
-		<artifactId>org.osgi.annotation</artifactId>
-		<version>${version.org.osgi.annotation}</version>
-	</dependency>
-	<dependency>
-		<groupId>org.osgi</groupId>
-		<artifactId>org.osgi.service.metatype.annotations</artifactId>
-		<version>${version.org.osgi.service.metatype.annotations}</version>
-	</dependency>
-	<dependency>
-		<groupId>org.osgi</groupId>
-		<artifactId>org.osgi.compendium</artifactId>
-		<version>${version.org.osgi.compendium}</version>
-		<scope>provided</scope>
-	</dependency>
+            <dependency>
+                <groupId>com.sun.xml.bind.wso2</groupId>
```

Comment:
This dependency is already available in the pom

Reply:
Removed